tagmanager/ctags/ctags.c

```diff
@@ -49,6 +49,7 @@
 
 
 
+#ifndef GLIB_MAJOR_VERSION
```

Comment:
I'm all for deleting as much code as possible :) If the block is left, IMO it should be changed to `#if 0` since it's obvious that it's more like a comment, whereas using `GLIB_MAJOR_VERSION` is kind of confusing.